package.json

```diff
@@ -34,11 +34,11 @@
   },
   "private": true,
   "dependencies": {
-    "@alfresco/adf-content-services": "6.9.0-8694813215",
-    "@alfresco/adf-core": "6.9.0-8694813215",
-    "@alfresco/adf-extensions": "6.9.0-8694813215",
-    "@alfresco/eslint-plugin-eslint-angular": "6.9.0-8694813215",
-    "@alfresco/js-api": "7.8.0-8694813215",
+    "@alfresco/adf-content-services": "6.9.0-8705449785",
+    "@alfresco/adf-core": "6.9.0-8705449785",
+    "@alfresco/adf-extensions": "6.9.0-8705449785",
+    "@alfresco/eslint-plugin-eslint-angular": "6.9.0-8705449785",
+    "@alfresco/js-api": "7.8.0-8705449785",
```

Comment:
`"@alfresco/adf-cli"` should be bumped as well?

Reply:
good point, updated